invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/deposit.html

```diff
@@ -15,6 +15,7 @@
 {%- extends config.BASE_TEMPLATE %}
 
 {%- block page_body %}
+<main id="main">
```

Comment:
This is fine by me. I wonder though if we can place it above this line in here: https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/templates/semantic-ui/invenio_theme/page.html#L70, so that every Jinja page gets it right away.  You can make an issue and investigate it after.

Reply:
This does seem to work and is a much better solution!
Confirming this is ok to do? I don't really have a sense of what other projects invenio_theme is used for.

Reply:
So like this: https://github.com/inveniosoftware/invenio-theme/compare/master...lhenze:a11y-page-landmarks?expand=1

Reply:
Just like that. Nice! You can submit that new one as a PR instead.